### PR TITLE
DOCS-1733 Add go runtime metrics page

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds page to Runtime Metrics docs about Go runtime metrics

### Motivation
DOCS-1733

### Preview

https://docs-staging.datadoghq.com/kari/docs-1733-go-runtime-metrics/tracing/runtime_metrics/go

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
Technical review requested in #apm-go

**Review by gbbr — APPROVED** [review]
Very nice job! Thanks!

File: content/en/tracing/runtime_metrics/go.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- **ECS**: [Set the appropriate flags in your task definition][5].
+
+## Data Collected
+
+The following metrics are collected by default after enabling Go metrics.
+
+{{< get-metrics-from-git "go" >}}
+
+<!-- Along with displaying these metrics in your APM Service Page, Datadog provides a [default Go Runtime Dashboard][?] with the `service` and `runtime-id` tags that are applied to these metrics.
+-->
```
Comment (kayayarai) [inline content/en/tracing/runtime_metrics/go.md:44]:
```suggestion

```

File: content/en/tracing/runtime_metrics/go.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      tag: 'Documentation'
+      text: 'Implement Opentracing across your applications.'
+    - link: 'tracing/visualization/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+## Automatic configuration
+
+Enable Go runtime metrics collection, start the tracer using the `WithRuntimeMetrics` option:
```
Comment (apigirl) [inline content/en/tracing/runtime_metrics/go.md:22]:
```suggestion
To enable Go runtime metrics collection, start the tracer using the `WithRuntimeMetrics` option:
```

**Review by apigirl — APPROVED** [review]
just one tiny feedback - looks great!

**Pull request merged**